numpy/ma/core.py

```diff
@@ -880,7 +880,17 @@ def __call__(self, x):
             return umath.less(x, self.critical_value)
 
 
-class _MaskedUnaryOperation(object):
+class _MaskedUFunc(object):
+    def __init__(self, ufunc):
+        self.f = ufunc
+        self.__doc__ = ufunc.__doc__
+        self.__name__ = ufunc.__name__
+
+    def __str__(self):
+        return "Masked version of {}".format(self.f)
```

Comment:
No idea why this isn't `__repr__`, but this at least maintains the old behaviour